api/app.py

```diff
@@ -30,7 +30,7 @@
 @app.before_first_request
 def initialize_database(autoinitialize=True):
     testing = 'TESTING' in os.environ
-    db_url = Configuration.database_url(testing)
+    db_url = Configuration.database_url()
```

Comment:
This is an unrelated change caused by the Configuration refactoring, necessary to get the app server to start. I believe this change made it into `master` after the `audiobooks` branch was created.